### DISCUSSION TITLE
ERROR: The QUANTITY AMPUA_Z requires liquid droplets when calculating [Species: SOOT] Accumulated Mass Per Unit Area

### DISCUSSION THREAD

**Discussion by ziuzu** [firemodels/fds#13286] [Q&A]
I am having trouble running the simulation with the [Species: SOOT] Accumulated Mass Per Unit Area statistic measurement. When I start the simulation, an error appears: "ERROR: The QUANTITY AMPUA_Z requires liquid droplets". I couldn't find information on how to add liquid droplets in the manual.

The simplified model consists of a fire, a supply vent, and an open vent – so that the soot content in the room changes.

Any tips on what I might be doing wrong that the simulation won't start?

fds:
[v2.zip](https://github.com/user-attachments/files/16509420/v2.zip)

**Comment by drjfloyd** [comment]
There is an entire chapter in the User's Guide on droplets; however, a bigger issue is that SOOT is not a liquid and trying to treat it as a liquid species would not be appropriate.  I recommend you carefully review the entire User's Guide and Technical Reference Guide. They contain multiple examples and theory discussions on modeling soot behavior as a paritlce.


<img width="427" alt="image" src="https://github.com/user-attachments/assets/b7daa766-4efa-4c8d-b6cb-ec3018e7d07e">
